setup.sh

```diff
@@ -91,3 +90,27 @@ else
     echo "genderize.io reference data found"
 fi
 
+# get LSTM model for name origin prediction
+# check for it first
+echo "Checking for LSTM name model..."
+if [[ ! -f ./name_lstm_models/LSTM.h5 ]]; then
+    read -r -p "LSTM.h5 model was not found, download? [y/N] " response
+    case "$response" in
+        [yY][eE][sS]|[yY])
+            curl -L "https://github.com/greenelab/wiki-nationality-estimate/raw/master/models/NamePrism.h5" \
```

Comment:
This may not be relevant anymore, since we're not updating wiki-nationality-estimate very often these days, but I was previously advised to point to a specific commit of the model, i.e. `raw/7425af1021f8a5c00aad789ebcaef67c5fe427bb/` instead of `raw/master/`. Then you can confirm that the output is consistent across runs. Up to you.

Reply:
this is a good suggestion, I will add it!

Reply:
added in https://github.com/greenelab/nature_news_disparities/pull/21/commits/4c356fa73825280d28af012cb53d0b7765a408cc